index.js

```diff
@@ -1,66 +1,100 @@
 'use strict';
 
 const app = require('express')();
+const hof = require('hof');
 const churchill = require('churchill');
 const path = require('path');
-const hofMiddleware = require('hof').middleware;
+const http = require('http');
+const https = require('https');
 const router = require('./lib/router');
 const serveStatic = require('./lib/serve-static');
 const sessionStore = require('./lib/sessions');
 const settings = require('./lib/settings');
 const defaults = require('./lib/defaults');
+const logger = require('./lib/logger');
 
 const getConfig = function getConfig() {
   const args = [].slice.call(arguments);
   return Object.assign.apply(null, [{}, defaults].concat(args));
 };
 
-module.exports = options => {
+const loadRoutes = config => {
+  config.routes.forEach(route => {
+    const routeConfig = Object.assign({}, {route}, config);
+    app.use(router(routeConfig));
+  });
+};
 
-  const load = (config) => {
-    config.routes.forEach((route) => {
-      const routeConfig = Object.assign({}, {route}, config);
-      app.use(router(routeConfig));
-    });
-  };
+const applyErrorMiddlewares = (config, i18n) => {
+  app.use(hof.middleware.notFound({
+    logger: config.logger,
+    translate: i18n.translate.bind(i18n),
+  }));
+
+  app.use(hof.middleware.errors({
+    translate: i18n.translate.bind(i18n),
+    debug: config.env === 'development'
+  }));
+};
+
+const getI18n = (config) =>
```

Comment:
IMO, it is easier to manage because we use it in the `start` function; https://github.com/UKHomeOffice/hof-bootstrap/pull/46/files#diff-168726dbe96b3ce427e7fedce31bb0bcR65, otherwise we'd need to pass it into `start`, which becomes very tricky when the user calls `start`